aspnetcore/grpc/browser.md

```diff
@@ -20,37 +20,21 @@ It is not possible to call a HTTP/2 gRPC service from a browser-based app. [gRPC
 
 ## Configure gRPC-Web in ASP.NET Core
 
-gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services, the only modification is startup configuration.
+gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services. The only modification is startup configuration.
 
 To enable gRPC-Web with an ASP.NET Core gRPC service:
 
 1. Add a reference to the [Grpc.AspNetCore.Web](https://www.nuget.org/packages/Grpc.AspNetCore.Web) package.
 2. Configure the app to use gRPC-Web by adding `AddGrpcWeb(...)` and `UseGrpcWeb()` to *Startup.cs*:
 
-```csharp
-public void ConfigureServices(IServiceCollection services)
-{
-    services.AddGrpc();
-}
+[!code-csharp[](~/grpc//browser/sample/Startup.cs?name=snippet_1&highlight=4,11,15)]
 
-public void Configure(IApplicationBuilder app)
-{
-    app.UseRouting();
+The preceding code:
 
-    // Add gRPC-Web middleware after routing and before endpoints
-    app.UseGrpcWeb();
+* Adds the gRPC-Web middleware, `UseGrpcWeb`, after routing and before endpoints.
+* Specifies the `endpoints.MapGrpcService<GreeterService>()` method supports gRPC-Web with `EnableGrpcWeb()`. Alternatively, configure all services to support gRPC-Web by adding `services.AddGrpcWeb(o => o.GrpcWebEnabled = true);` to ConfigureServices.
```

Comment:
`endpoints.MapGrpcService<GreeterService>()` is OK and an exception to the rule. NEVER keep `()` in functions. It should be
`EnableGrpcWeb` not
`EnableGrpcWeb()`